src/ethereum/EnigmaContractAPIBuilder.js

```diff
@@ -200,37 +211,36 @@ class EnigmaContractAPIBuilder {
    * */
   async setConfigAndBuild(options) {
     let res;
-    let ethereumAddress = null;
-
-    // urlProvider: this._ethereumWebsocketProvider,
-    // enigmaContractAddress: this._enigmaContractAddress,
-    // accountAddress: this._ethereumAddress,
-    // enigmaContractAbiPath: this._enigmaContractAbiPath,
-    // accountKeyPath: this._ethereumKeyPath
+    let config = {};
 
-    if (options.accountAddress) {
-      ethereumAddress = options.accountAddress;
-    }
-
-    if (options.accountKey) {
-      this.accountKey = options.accountKey;
-    }
+    const operationalAddress = options.accountAddress ? options.accountAddress : null;
```

Comment:
We should probably change the names of the options as well. Would be less confusing, and also if we match the names of the variables with the options we could use object destructing:

`const {accountAddress, operationalKey, stakingAddress, minimunConfirmations} = options`